mobile/src/screens/chat/libs/helpers/message-item-properties.helper.ts

```diff
@@ -0,0 +1,58 @@
+import { TimeFormat } from '#libs/enums/enums';
+import { getFormattedDate } from '#libs/helpers/helpers';
+import { type ChatMessageGetAllItemResponseDto } from '#packages/chat-messages/chat-messages';
+
+import { NEXT_USER, PREVIOUS_USER } from '../constants/constants';
+
+type Properties = {
+  messages: ChatMessageGetAllItemResponseDto[];
+  currentIndex: number;
+};
+
+type MessageItemProperties = {
+  isTimeVisible: boolean;
+  isDifferentMessageOwner?: boolean;
+  currentTime?: string;
+};
+
+const messageItemProperties = ({
```

Comment:
I suppose we can group messages on backend side https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/345

Reply:
In my opinion we are able not to implement it for now because of bot can't send more than one message